void tomo::IoLoop::run_loop<true, false, false, true, false>()
--- PRE

+++ POST

@@ -545,7 +545,7 @@

 0a5c [ 4] nopl 0x0(%rax)
 0a60 [ 5] mov $0x1,%esi
 0a65 [ 3] mov %rbx,%rdi
-0a68 [ 5] call <tomo::IoLoop::arm_accept(tomo::UrKind)>
+0a68 [ 5] call <tomo::IoLoop::fliplisten_retry_accept(tomo::UrKind)>
 0a6d [ 5] jmp <+0x2e2>
 0a72 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0a7d [ 3] nopl (%rax)
